programs/CMakeLists.txt

```diff
@@ -5,3 +5,4 @@ add_subdirectory( eosio-launcher )
 add_subdirectory( eosio-blocklog )
 add_subdirectory( nodeos-sectl )
 add_subdirectory( rodeos )
+add_subdirectory( eosio-tester )
```

Comment:
Can you make it optional in the same way for rodeos? Default is not to build it.